origin-js/src/contractInterface/users/v00_adapter.js

```diff
@@ -116,7 +117,7 @@ class V00_UsersAdapter {
             return data.substr(2)
           })
           .join('')
-      const dataOffsets = attestations.map(() => 32) // all data hashes will be 32 bytes
+      const dataOffsets = attestations.map(({ data }) => (data.length - 2) / 2) // deduct 2 because '0x' is removed
```

Comment:
nit: it is not completely obvious so maybe add a comment to explain the calculation of data offset. Something along the lines that binary data for each attestation is passed as string using hexadecimal encoding. For ex: "0x12AB03". To get the binary data length, we subtract the "0x" prefix and divide the reminder string length by 2.

Reply:
👍